django_babel/extract.py

```diff
@@ -1,5 +1,14 @@
 # -*- coding: utf-8 -*-
-from django.template.base import Lexer, TOKEN_TEXT, TOKEN_VAR, TOKEN_BLOCK
+try:
+    from django.template.base import Lexer, TOKEN_TEXT, TOKEN_VAR, TOKEN_BLOCK
+except ImportError:
+    # Django 2.1+
+    from django.template.base import Lexer, TokenType
+
+    TOKEN_TEXT = TokenType.TEXT
+    TOKEN_VAR = TokenType.VAR
+    TOKEN_BLOCK = TokenType.BLOCK
+
```

Comment:
I think the general recommendation is rather to `try` new imports, and `except` to fallback on old imports:

```
try:
    from django.template.base import Lexer, TokenType
except ImportError:  # django < 2.1
    from django.template.base import Lexer, TOKEN_TEXT, TOKEN_VAR, TOKEN_BLOCK
else:
    TOKEN_TEXT = TokenType.TEXT
    TOKEN_VAR = TokenType.VAR
    TOKEN_BLOCK = TokenType.BLOCK
```

Reply:
@jpic that makes sense, thank you!

Reply:
np, keep up the gr8 work :tophat: